tests/queries/0_stateless/02377_optimize_sorting_by_input_stream_properties_explain.reference

```diff
@@ -6,13 +6,13 @@ Sorting (Global): a ASC
 Sorting (None)
 Sorting (None)
 -- disable optimization -> sorting order is NOT propagated from subquery -> full sort
--- QUERY: set optimize_sorting_by_input_stream_properties=0;set max_threads=1;EXPLAIN PIPELINE SELECT a FROM (SELECT a FROM optimize_sorting) ORDER BY a
+-- QUERY: set optimize_sorting_by_input_stream_properties=0;set query_plan_read_in_order=0;set max_threads=1;EXPLAIN PIPELINE SELECT a FROM (SELECT a FROM optimize_sorting) ORDER BY a
 MergeSortingTransform
 LimitsCheckingTransform
 PartialSortingTransform
 -- enable optimization -> sorting order is propagated from subquery -> merge sort
 -- QUERY: set optimize_sorting_by_input_stream_properties=1;set max_threads=1;EXPLAIN PIPELINE SELECT a FROM (SELECT a FROM optimize_sorting) ORDER BY a
-MergeSortingTransform
+MergingSortedTransform 3 → 1
```

Comment:
From the comment, there should have been `MergingSortedTransform` initially. `MergeSortingTransform` is a full sort.
It was magically fixed.